go.mod

```diff
@@ -255,4 +248,6 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.41.0-tm-v0.34.29
+
+	nhooyr.io/websocket => nhooyr.io/websocket v1.8.17
```

Comment:
I had to bump the version for the websocket as there was a complain on not finding the `v1.8.6`

Reply:
Is it not possible to do without the replace statement (I.e. can we bump it directly?)

Reply:
It is indirect and used in some dependencies, so I am not sure if that's possible. Do you know if we can do that?

Reply:
maybe try changing line 236 to use the new version?

Reply:
BTW, this `nhooyr.io/websocket` is not used in `knuu` nor the `e2e test`, but not sure why it did not find it on my machine.
```sh
go mod why nhooyr.io/websocket
# nhooyr.io/websocket
github.com/celestiaorg/celestia-app/v3/cmd/celestia-appd/cmd
github.com/cosmos/cosmos-sdk/server/grpc
github.com/improbable-eng/grpc-web/go/grpcweb
nhooyr.io/websocket
```

there might be something wrong with my `goproxy`, could you please remove it and then test it locally to see if everything is alright; so we can remove that replace entirely?
thanks